prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,36 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.core.window import Window
+
+FACTOR_MILES_TO_KM = 1.60934
+
+
+class MilesConverterApp(App):
+    """Convert miles to kilometres"""
+
+    def build(self):
+        """Build the Kivy app"""
+        Window.size = (400, 200)
+        self.title = "Convert Miles to Kilometres"
+        self.root = Builder.load_file('convert_miles_km.kv')
+        return self.root
+
+    def handle_calculate(self, miles):
+        """Calculate """
+        calculation = self.get_valid_number() * FACTOR_MILES_TO_KM
+        self.root.ids.output_label.text = str(f"{calculation}")
+
+    def handle_increment(self, increment):
```

Comment:
Your handle_increment code does not work currently because it expects the parameter input_miles to be passed in.